srcpkgs/verilator/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'foo'
+pkgname=verilator
+version=4.102
+revision=1
+build_style=configure
+configure_args="--prefix /usr"
+hostmakedepends="autoconf"
+makedepends="flex"
+short_desc="C++ Library for RTL emulation"
```

Comment:
Why not use verilator's official description, "SystemVerilog simulator and lint system"